crates/svg2pdf-cli/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "svg2pdf-cli"
+description = "The command line interface for svg2pdf."
+categories = ["encoding", "graphics", "multimedia", "command-line-utilities"]
+keywords = ["svg2pdf", "cli"]
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+repository.workspace = true
+license.workspace = true
+build = "build.rs"
+
+[[bin]]
+name = "svg2pdf"
+path = "src/main.rs"
+test = false
+doctest = false
+bench = false
+doc = false
+
+[dependencies]
+svg2pdf = { path = "../svg2pdf" }
```

Comment:
I'm not sure whether this will work with crates.io. Typst isn't yet on there, so it's not necessarily taken care of.

Reply:
You have a good point. What do you think is best, leaving it as is for now, or increasing the minor version and referencing that on the CLI?

Reply:
We can just leave it like that for now and I'll take care of it before the next release.